utils/build-script-impl

```diff
@@ -454,6 +454,10 @@ function set_build_options_for_host() {
             SWIFT_HOST_TRIPLE="aarch64-unknown-linux-android"
             llvm_target_arch="AArch64"
             ;;
+        android-armv7)
+            SWIFT_HOST_TRIPLE="armv7-unknown-linux-androideabi"
```

Comment:
While [the Android NDK says to use `armv7a` as the architecture in the triple](https://developer.android.com/ndk/guides/other_build_systems), I see that [LLVM considers `armv7` a synonym](https://github.com/llvm/llvm-project/blob/f16b2d83154aed71aaf9a0717fbb0199d027f312/llvm/lib/Support/ARMTargetParser.cpp#L146) and I don't see any code in LLVM, clang, or Swift that treats them differently. I've been using both interchangeably and the only problem I had was that [SPM only recognizes `armv7`](https://github.com/termux/termux-packages/pull/5843/files#diff-49d3fe0505b12cff1f474dc64b78d001), so use that here too.

Reply:
Seems reasonable.  `armv7` and `armv7-a` are synonyms, the A cores are the default.